### ISSUE TITLE
DEV: LOG: Create the log object based on the response model from SLB

### ISSUE THREAD

**Issue by cherrera2001** [hashmapinc/Drillflow#316]

**Comment by niaalex** [comment]
this item is no longer blocked/in progress

**Comment by niaalex** [comment]
In Progress ETA 4/26 with AddtoStore

**Comment by TessForGithub2** [comment]
1. unmarshal incoming JSON String that represents channelSet + channels as a response from DoT
2. now the POJOs are populated from the unmarshal operation, so perform the transformations per mapping document to return a JSONObject ready for transformation to WITSML XML (performed by the caller before returning as XMLOUT)

I have completed the unmarshal; I will test once Sukhe puts in the fix I requested last night (4/29) so he is sending me more than channels (a JSONArray) but a JSONObject as described above.

I will continue on with mapping & writing JUnit testing against this module. The mapping activity itself is documented in card #381 which is connected to branch #364.
